src/v0/destinations/tune/transform.js

```diff
@@ -43,10 +50,13 @@ const responseBuilder = (message, { Config }) => {
       ...mapPropertiesWithNestedSupport(message, properties, advUniqueIdHashMap),
     };
 
+    const endpoint = getTuneEndpoint(subdomain);
+    const queryString = toQueryString(params);
+    const finalEndpoint = queryString ? `${endpoint}?${queryString}` : endpoint;
```

Comment:
This is not needed as server adds the `params` as query params to url by default

Reply:
@utsabc but then shouldn't it get reflected in the test case ?

Reply:
we dont need that, server will handle from params you can test once in local

Reply:
done